packages/sdks/core-js-sdk/test/index.test.ts

```diff
@@ -25,17 +24,20 @@ describe('sdk', () => {
     expect(createSdk({ projectId: '123' })).toBe('sdk');
     expect(sdk).toHaveBeenCalledWith('httpClient');
     expect(httpClient).toHaveBeenCalledWith({
-      baseUrl: DEFAULT_BASE_API_URL,
+      baseUrl: undefined,
```

Comment:
Maybe we should add a few tests to the HTTP client to verify the logic that was migrated
WDYT?